newsfragments/3507.bugfix.rst

```diff
@@ -0,0 +1 @@
+Invite without HumanHandle should not crash
```

Comment:
This message is oriented towards users and will appear in the change log (see https://docs.parsec.cloud/en/latest/history.html)
Use a verb like `Fix ...` and mention that the issue only affected legacy users.